docs/tutorials/360-lookaround-camera.md

```diff
@@ -0,0 +1,9 @@
+---
+title: '360 lookaround camera'
+tags: [input, camera, tutorial]
+thumb: https://s3-eu-west-1.amazonaws.com/images.playcanvas.com/projects/12/438216/3B51C6-image-75.jpg
+---
```

Comment:
The new Markdown files need some tweaks for better whitespacing. This is because Transifex breaks up Markdown into sub-strings for easier translation. And without the whitespacing, it's all lumped into the same string. So a blank like is needed after the `---` here...

Reply:
Fixed now!